### PR TITLE
Translation 1819 translations mt5 ib

### PR DESCRIPTION
**Pull request by eduard-deriv**
Changes:

-  ib commissions mt5 cards updates, added mt5 swapFree card.

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/partners/affiliate-ib/_dmt5-cards.tsx [hunk 1]
```diff
@@ -76,33 +76,82 @@ const query = graphql`
...
-        @media ${device.tabletL} {
-            margin: 2.4rem 0;
-        }
-    }
-
     @media ${device.laptopM} {
-        min-width: 38.4rem;
-        width: 38.4rem;
+        min-inline-size: 38.4rem;
+        inline-size: 38.4rem;
```
Comment (aswathy-deriv) [inline src/pages/partners/affiliate-ib/_dmt5-cards.tsx:150]: Is both `min-inline-size` and` inline-size` needed?

Comment (eduard-deriv) [reply src/pages/partners/affiliate-ib/_dmt5-cards.tsx:150]: no, just renamed the existing styles correctly, remove then

File: src/pages/partners/affiliate-ib/_dmt5-cards.tsx [hunk 2]
```diff
@@ -112,11 +161,12 @@ const StyledCard = styled(Card)`
     }
 
     @media ${device.mobileM} {
-        min-width: unset;
-        width: 100%;
+        min-inline-size: unset;
+        inline-size: 100%;
```
Comment (aswathy-deriv) [inline src/pages/partners/affiliate-ib/_dmt5-cards.tsx:165]: same here

**Review by konstantinosG-deriv — APPROVED** [review]

File: src/pages/partners/affiliate-ib/_dmt5-cards.tsx [hunk 3]
```diff
@@ -131,7 +181,7 @@ const CardText = styled(Text)`
 `
 const AccordionWrapper = styled.div`
     max-width: 99.6rem;
-    width: fit-content;
+    width: 100%;
```
Comment (jia-deriv) [inline src/pages/partners/affiliate-ib/_dmt5-cards.tsx:184]: What is the different between fit-content and 100% width here?

**Review by hasan-deriv — APPROVED** [review]

File: src/pages/partners/affiliate-ib/_dmt5-cards.tsx (hunk 1 above)
Comment (eduard-deriv) [reply src/pages/partners/affiliate-ib/_dmt5-cards.tsx:150]: remove

File: src/pages/partners/affiliate-ib/_dmt5-cards.tsx (hunk 2 above)
Comment (eduard-deriv) [reply src/pages/partners/affiliate-ib/_dmt5-cards.tsx:165]: remove

File: src/pages/partners/affiliate-ib/_dmt5-cards.tsx (hunk 3 above)
Comment (eduard-deriv) [reply src/pages/partners/affiliate-ib/_dmt5-cards.tsx:184]: this has been changed to match the design due to the addition of a fourth element and the division of elements into 2 blocks, width 100% not work correctly with new approach

**Review by jia-deriv — APPROVED** [review]

**Review by aswathy-deriv — DISMISSED** [review]

**Review by aswathy-deriv — APPROVED** [review]

**Pull request merged**